CONTRIBUTING.md

```diff
@@ -32,43 +38,24 @@ plugins:
       local: </path/to/repo/root>
 ```
 
-Adding a plugin source lets users run `trunk check enable` or `trunk actions enable` with linters
-and actions defined in that plugin. For more information, see our
+Adding a plugin source lets users run `trunk check enable`, `trunk tools enable`, or
+`trunk actions enable` with definitions in that plugin. For more information, see our
 [docs](https://docs.trunk.io/docs/plugins).
 
+**Please review our [Testing Guide](tests/README.md) for info on writing and running tests.**
+
 If you have questions, please [stop by our community Slack](https://slack.trunk.io/), and if you
 have a feature request, you can [file it here](https://features.trunk.io/).
 
-## Releases
-
-`trunk-io/plugins` is released on a fairly regular cadence that is independent of PRs. Users will
-pick up these configuration changes by running `trunk upgrade` to automatically update their plugin
-version to the latest release.
-
-```yaml
-plugins:
-  sources:
-    - id: trunk
-      uri: https://github.com/trunk-io/plugins
-      ref: v1.2.5 # will change to the latest release on next `trunk upgrade`
-```
-
-We recommend only setting the above `ref` field to be our released tags, but if you need a linter or
-action that hasn't been released yet, you can set `ref` to be a git SHA. Do **not** set `ref` to be
-a branch name, or `HEAD`, as users will observe buggy behavior.
-
-Note that the ref and the cli version in the trunk.yaml must be compatible. This is managed by
-`required_trunk_version`, as specified in [`plugin.yaml`](plugin.yaml). Users will not be able to
-load a plugin source until they have upgraded to a compliant CLI version.
-
 ## Linters
 
 To add a new linter:
 
 1. Run `trunk check` to start up Trunk in the background.
-2. Create a directory inside `linters/` with the name of your new linter.
-3. Inside this new directory, create the following structure. Most of these files will be
-   automatically created for you:
+2. Run `mkdir linters/<my-linter>` to start. This should autopopulate with a sample
```

Comment:
Do you have to run this before check finishes? I think this is the part I missed 😆

Reply:
Really `trunk check` here is just a shorthand for launching our daemon, which listens for file changes in order to run [Trunk Actions](https://github.com/trunk-io/plugins/blob/main/.trunk/trunk.yaml#L47-L54). `trunk check` should finish almost immediately if you're starting from a fresh checkout.

Running `mkdir` after that (1-2 seconds after the `trunk check` command starts) should be sufficient. But if you ran something like `code linters/vale/plugin.yaml`, the [script](https://github.com/trunk-io/plugins/blob/main/repo-tools/linter-test-helper/generate#L80-L85) won't autogen since it only checks empty directories :)

Reply:
We have `trunk daemon start` that just stats the daemon.